Resources/doc/features/headers.rst

```diff
@@ -69,5 +69,25 @@ This is an example configuration. For more, see the
                         cache_control: { public: true, max_age: 15, s_maxage: 30 }
                         etag: "strong"
 
+This is an example with the special `` and `` configurations.
```

Comment:
sorry, i notice there are 2 headers.rst files in the project.

i meant to add to the other one Resources/doc/reference/configuration/headers.rst

that one has a section specifically on the reverse proxy ttl configuration.

Reply:
got it, will move it there